### ISSUE TITLE
Homegear 0.7 Raspbian Image Download (Jessie) doesn't work out of the box

### ISSUE THREAD

**Issue by b-bittner** [Homegear/Homegear#296]
When installing the image, fire up the board and answering all questions (with default), homegear will not be on the installation.

After some searching and debugging I found this:
```
pi@homegear:~$ sudo apt-get -y install homegear homegear-nodes-core homegear-nodes-extra homegear-homematicbidcos homegear-homematicwired homegear-insteon homegear-max homegear-philipshue homegear-sonos homegear-kodi homegear-ipcam homegear-beckhoff homegear-knx homegear-enocean homegear-intertechno
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 homegear-beckhoff : Depends: libhomegear-base (= 0.7.10-1397) but 0.7.12-1492 is to be installed
                     Depends: homegear (= 0.7.10-1397) but 0.7.12-1492 is to be installed
 homegear-enocean : Depends: libhomegear-base (= 0.7.10-1397) but 0.7.12-1492 is to be installed
                    Depends: homegear (= 0.7.10-1397) but 0.7.12-1492 is to be installed
 homegear-knx : Depends: libhomegear-base (= 0.7.10-1397) but 0.7.12-1492 is to be installed
                Depends: homegear (= 0.7.10-1397) but 0.7.12-1492 is to be installed
E: Unable to correct problems, you have held broken packages.
```

it seems there are some broken dependencies.

To reproduce the problem, just try to install homegear on a new SD-Card.

**Comment by hfedcba** [comment]
Thanks and yes there are broken dependencies. It should be fixed by commit c8e421534b30d9cfdbab59679304e8ddd61b65e8. I'm testing it right now. I removed version 0.7.10 for now so it should work again with version 0.7.9. Tomorrow 0.7.12 should be released finally fixing the problem.

**Comment by hfedcba** [comment]
Works again! It'll take a few days for 0.7.12 to compile.
